### DISCUSSION TITLE
Planted Trees can grow into other's claims

### DISCUSSION THREAD

**Discussion by kitokofox** [GriefPrevention/GriefPrevention#152] [Ideas]
### What behaviour is expected:
Tree slices off/ not allowed to grow.
### What behaviour is observed:
Tree grew leaves into other's claim.
### Steps/models to reproduce:
Plant a tree adjacent to other's claim and then let it grow.

### Spigot/Craftbukkit/Paper and GriefPrevention version:
This can be found on any version.

**Comment by RoboMWM** [comment]
I remember seeing something in the commits about removing this since it's not that big of a deal & somewhat expensive(?)

**Reply by RoboMWM** [reply]
ah, this got transferred here because it must've been tagged as addon suggestion as other than not being checked, it was assumed there was no issue with this. Superceded by #1116 which claims that tile entities can be broken due to this.
